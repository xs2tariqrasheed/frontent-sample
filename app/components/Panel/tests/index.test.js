import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import { Panel, PanelBody, PanelHeader } from '../index';

describe('<Spinner />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <Panel isLeftAligned={false}>
          <PanelHeader isLight>Header</PanelHeader>
          <PanelBody>Hello</PanelBody>
        </Panel>
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
