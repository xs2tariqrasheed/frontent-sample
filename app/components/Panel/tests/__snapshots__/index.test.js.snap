// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render and match the snapshot 1`] = `
<div
  class="Panel-sc-44axhw-0 elyGcO"
>
  <div
    class="Panel__PanelHeader-sc-44axhw-1 gwYQyg"
  >
    Header
  </div>
  <div
    class="Panel__PanelBody-sc-44axhw-2 isMgK"
  >
    Hello
  </div>
</div>
`;
